PipelineRunnerExtension/README.MD

```diff
@@ -0,0 +1,63 @@
+# Configurable Pipeline Runner
+
+The purpose of this Build task is to provide a solution for :
+
+- Triggering Remote/Delegate/Child Pipelines any amount of times from a Parent pipeline.
+- The build task allows passing variables that are passed to the delegate pipeline at runtime.
+- The variable array in Json Array format will determine how many delegate pipeline instances are started (Restricted to the Pipeline Agents Available)
```

Comment:
What happens if you exceed the number of Pipeline Agents Available? Will the parent pipeline fail, or will it slow down the build?

Reply:
It will queue the excess ones until there is an agent available, while the timeout value is within the expiration bounds